packages/core/src/shared/crashMonitoring.ts

```diff
@@ -453,8 +490,14 @@ export class FileSystemState {
 
         // Clean up the running extension file since it is no longer exists
         const dir = await this.runningExtsDir()
-        // Use force since another checker may have already removed this file before this is ran
-        await withFailCtx('deleteStaleRunningFile', () => fs.delete(path.join(dir, extId), { force: true }))
+        // Retry on failure since failing to delete this file will result in incorrectly reported crashes.
+        // The common errors we were seeing were windows EPERM/EBUSY errors. There may be a relation
+        // to this https://github.com/aws/aws-toolkit-vscode/pull/5335
+        await withRetries(() => withFailCtx('deleteStaleRunningFile', () => fs.delete(path.join(dir, extId))), {
+            maxRetries: 7,
```

Comment:
were this numbers determined by something?

Reply:
This just equated to 12.7 seconds at most which I think is enough gracetime for the fs operation to succeed. But I think it may be worth to double it to 25 seconds looking back at it